app/models/costume.rb

```diff
@@ -1,2 +1,14 @@
 class Costume < ApplicationRecord
+  CATEGORIES = ["superhero", "historical", "movie", "tv show", "animal", "other"]
+  SIZES = ["S", "M", "L", "XL", "XXL", "other"]
+
+  # validations 
+  validates :name, :category, :size, :description, :price_per_day, presence: true
+  validates :name,  :description, uniqueness: true
+  validates :name, :description, length: { minimum: 5 }
+  validates :price_per_day, numericality: { greater_than: 0 }
+  validates :category, inclusion: { in: CATEGORIES }
+  validates :size, inclusion: { in: SIZES }
```

Comment:
Not sure if these will be necessary since the users will probably choose from a dropdown menu or anything with prefixed options.

Reply:
You can use the categories and sizes array in the form builder for the drop down.

Reply:
Hmm I thought it would be more visible that I was commenting on a specific line. I meant the validations for size and categories. In my head the validation would only be necessary if the user was to type those specific words or letters from the arrays themselves. If they choose them from a drop down, they can't really enter invalid information because they have to choose from the ones we put in the drop down.